content/blog/2020/11/2020-11-11-jira-upgrade-for-the-jenkins-project.adoc

```diff
@@ -0,0 +1,134 @@
+The Linux Foundation uses the DNS method to obtain SSL certificates.
+Unfortunately, the Jenkins project uses the HTTP method to obtain SSL certificates.
+
+Thankfully, Olivier Vernin of the Jenkins project and Anton Baranov of the Linux Foundation found a solution.
+They were able to grant sufficient permissions in the Jenkins DNS records so that the Linux Foundation could issue SSL certificates for issues.jenkins.io and issues.jenkins-ci.org.
+
+=== Building the Prototype
+
+Anton Baranov created a prototype Jira server, restored an older Jenkins Jira backup, and upgraded it to Jira 8.13.
+That first restore detected that we had not provided the Jira attachments or the Jira avatars.
+That attachments and avatars added multiple gigabytes to the initial backup data and were vital to complete the update.
+
+== Testing the Upgrade
+
+A group of volunteers including Jenkins users, security team members, and infrastructure team members tested the upgrade during the week of October 26, 2020.
+The tests confirmed that authentication worked as expected and that the Jira prototype was functioning as expected.
+
+We thank the test team, including:
+
+* Daniel Beck
+* Tim Jacomb
+* Olivier Vernin
+* Mark Waite
+
+The tests included:
+
+* Creating and routing issues
+* Commenting on issues
+* Viewing dashboards with the expected content
+* LDAP settings
+* Email notification
+
+The tests detected minor issues that Anton was able to correct in preparation for the final upgrade.
+The testing team agreed that the tests were successful.
+
+== Deploying the Upgrade
+
+Olivier Vernin announced the final upgrade by link:https://groups.google.com/g/jenkins-infra/c/i7SaB-24L1o/m/NeUFrH82AAAJ[email to the Jenkins infra list] with details of the changes happening during the upgrade.
+Monday, November 9, 2020, the final backup of the existing Jira server was copied into the new Linux Foundation server.
+
+The final upgrade encountered issues that we had not seen during the initial tests.
+The "bumps and bruises" from the unexpected issues were resolved by Anton Baranov as he used a multi-step upgrade process.
+The steps included:
+
+* Restore the earlier backup to Jira 7.13
+* Restore the most recent backup
+* Upgrade to Jira 8.13
+* Install avatars, attachments, and other images
+* Update DNS entries to point to the new Jenkins Jira server
+
+== Lessons from the Upgrade
+
+Lessons were related to timing, estimation, and communication.
+
+=== Scheduling the Upgrade
+
+The test upgrade started the week of October 19, 2020.
+It took several days longer than originally expected.
+Thankfully, we had allowed an extra week between the test upgrade and the production upgrade.
+
+The link:https://groups.google.com/g/jenkins-infra/c/8SC8bQuhD7s/m/Qtnz9o8gBAAJ[originally announced schedule] for final upgrade was intentionally placed in a week that would not include a long term support release.
```

Comment:
`for final upgrade was intentionally` should be `for the final upgrade was intentionally`